test/integration/upgrade-with-dependencies/00-install.yaml

```
@@ -0,0 +1,5 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: kubectl kudo install --instance operator ./operator-1.0
```

nit: it's somewhat confusing (though valid) to have an `Instance` with the name `operator`. 
```suggestion
  - command: kubectl kudo install --instance operator-instance ./operator-1.0
```
this will change the assert files